modules/seqexec/engine/src/main/scala/seqexec/engine/Execution.scala

```diff
@@ -98,15 +101,15 @@ object Result {
   // Base traits for results. They make harder to pass the wrong value.
   trait RetVal
   trait PartialVal
-  trait PauseContext
+  trait PauseContext[F[_]]
```

Comment:
This change makes it necessary to propagate `F[_]` up to several classes up the code tree